### ISSUE TITLE
Plugins development 

### ISSUE THREAD

**Issue by Awilum** [asmcss/assembler#11]
Hello! 

I can't find any examples of Plugins Development for ASMCSS as it is for Tailwind CSS.
For e.g. how should I develop and include such kind of plugin https://github.com/tailwindlabs/tailwindcss-line-clamp for ASMCSS? via separated js file ? js bundle ? css with mixin?

How to do this in the right way?

**Comment by msarca** [comment]
There is no need to create plugins and other complicated setups for such a simple task. You can easily achieve this by creating a [mixin](https://asmcss.com/docs/1.x/mixins.html). 

```html
<style>
:root {
    --line-clamp--mixin: "overflow:hidden; display:-webkit-box; box-orient:vertical; line-clamp:${0=1}";
}
</style>
```

Now you can use the newly created mixin.

```html
<div x-style="w:120px; ^line-clamp:4">
    Any veggie can belong on your menu, so choose what you love and use the rest for future meals and snacks you can enjoy in flavorful ways.
</div>
```

----

`box-orient` and `line-clamp` are supported starting with the newly released [v0.5.0](https://github.com/asmcss/assembler/releases/tag/0.5.0) version of AssemblerCSS. This version also brings users the ability to [register](https://asmcss.com/docs/1.x/configuration.html#data-register-properties) new [assembler properties](https://asmcss.com/docs/1.x/assembler-properties.html).

**Comment by Awilum** [comment]
okey, what about a little complex things ?
btw: I have create this plugin/mixin: https://github.com/Awilum/asmcss-mixin-w
